Scripts/generate-test-config.sh

```diff
@@ -198,6 +198,7 @@ cmd_quiet_flag="--quiet"
 [[ -n $region ]] || die "'region' not specified"
 export AWS_DEFAULT_REGION=${region}
 
+[[ ! -z $(which jq) ]] || die "jq is missing: brew install jq"
```

Comment:
The [`validate_dependencies` function](https://github.com/aws-amplify/aws-sdk-ios/pull/3517/files#diff-02697d3f7e59bcd8b9d7b6c10f286fbd108a92cb81d544a200f743df66577d3cL205) should already validate this. It's only necessary for the assume-role use case (which you should be using to generate the test config locally), so it isn't appropriate to validate every time. `validate_dependencies` is invoked as the [first step in `main`](https://github.com/aws-amplify/aws-sdk-ios/pull/3517/files#diff-02697d3f7e59bcd8b9d7b6c10f286fbd108a92cb81d544a200f743df66577d3cL298). If we're seeing failures, let's fix that invocation or function logic.

Reply:
Yes, it failed for me so I will look at that function to update it.

Reply:
I've determined this check will not be needed and removed it in the next commit.